src/components/datagrid/DataGrid.js

```diff
@@ -13,35 +13,21 @@ import HeaderRow from './HeaderRow';
 import BodyRow from './BodyRow';
 import cx from 'classnames';
 import omit from 'lodash.omit';
-import throttle from 'lodash.throttle';
+import ReactResizeDetector from 'react-resize-detector';
```

Comment:
Does the resizing and overflowing stuff belong to this PR?

Reply:
Yeah, you're right @lowiebenoot . I'll close this PR and split it up in two separate (and less messy) PRs.